packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -145,8 +144,146 @@ class InteractiveViewer extends StatefulWidget {
                  boundaryMargin.left.isFinite),
        ),
        constrained = false,
+       _transformChild = true,
        child = null;
 
+  /// Using `InteractiveViewer.raw` will prevent InteractiveViewer from transforming the
```

Comment:
You could turn this into a doc reference?

```suggestion
  /// Using [InteractiveViewer.raw] will prevent InteractiveViewer from transforming the
```